src/app/components/question-approval-page/question-approval-page.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { QuestionDto } from 'src/app/models/question-dto';
+import { QuestionService } from 'src/app/services/question.service';
+import { FormControl, FormGroup } from '@angular/forms';
+@Component({
+  selector: 'app-question-approval-page',
+  templateUrl: './question-approval-page.component.html',
+  styleUrls: ['./question-approval-page.component.scss']
+})
+export class QuestionApprovalPageComponent implements OnInit {
+  question: QuestionDto =  QuestionDto.createDtoWithNonEmptyFields();
+  exampleForm = new FormGroup({
+    sample: new FormControl(''),
+  });
+  constructor(
+    private questionService: QuestionService,
+  ) { }
+// maybe add put back button for concurrency issues. also add checks against approval 
```

Comment:
You can make use of the `// TODO` syntax as most IDEs look out for it. Additionally, it's worth raising this as an issue in this repo so we know it's something we'll need to fix eventually.